x-pack/plugins/cases/public/components/user_actions_activity_bar/filter_activity.tsx

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { EuiFilterGroup, EuiFilterButton } from '@elastic/eui';
+
+import type { CaseUserActionsStats } from '../../containers/types';
+import * as i18n from './translations';
+import type { FilterType } from './types';
+
+interface FilterActivityProps {
+  isLoading?: boolean;
+  type: FilterType;
+  userActionsStats?: CaseUserActionsStats;
+  onFilterChange: (type: FilterType) => void;
+}
+
+export const FilterActivity = React.memo<FilterActivityProps>(
+  ({ type, onFilterChange, userActionsStats, isLoading = false }) => {
+    const handleFilterChange = useCallback(
+      (value: FilterType) => {
+        if (value !== type) {
+          onFilterChange(value);
+        }
+      },
+      [onFilterChange, type]
+    );
+
+    return (
+      <EuiFilterGroup data-test-subj="user-actions-filter-activity-group">
+        <EuiFilterButton
+          withNext
+          grow={false}
+          onClick={() => handleFilterChange('all')}
+          hasActiveFilters={type === 'all'}
+          numFilters={userActionsStats && userActionsStats.total ? userActionsStats.total - 1 : 0} // subtracting user action of description from total
+          isLoading={isLoading}
+          data-test-subj="user-actions-filter-activity-button-all"
+        >
+          {i18n.ALL}
+        </EuiFilterButton>
+        <EuiFilterButton
+          withNext
+          grow={false}
+          hasActiveFilters={type === 'user'}
+          numActiveFilters={userActionsStats?.totalComments ?? 0}
```

Comment:
I think we should change `numActiveFilters` to `numFilters`. I tested and `numActiveFilters` do not show `0`. We should show the `0` counter to all filter options, right?

<img width="277" alt="Screenshot 2023-02-20 at 3 46 53 PM" src="https://user-images.githubusercontent.com/7871006/220154595-b1d5cb70-57c9-4365-a2f7-385f3fc611d4.png">

<img width="325" alt="Screenshot 2023-02-20 at 6 06 51 PM" src="https://user-images.githubusercontent.com/7871006/220154778-b7374716-01c9-41e4-8ec3-5e77b0822276.png">

Reply:
And regarding the number after `All`.

Could we perhaps have the same distance between the word and its count as in `Comments` and `History`?

<img width="316" alt="Screenshot 2023-02-21 at 11 38 25" src="https://user-images.githubusercontent.com/1533137/220322445-a620a974-7701-451d-90ec-30c9fce92c7e.png">

Maybe it is nitpicking but I find it weird that there is so much space there 😅

Reply:
haha.. Not sure how to fix it as it's an `elastic UI` component. But I can give it a try by playing around with it's `props` and `css`.

Reply:
Adjusted the All button size:

![image](https://user-images.githubusercontent.com/117571355/220597653-cce33d5f-48d4-40d7-86c2-a179fd30125d.png)

![image](https://user-images.githubusercontent.com/117571355/220597774-a8fae0ea-4895-4c79-89bf-30ac45958b72.png)